content/storing-in-cloud-must-try.md

```diff
@@ -0,0 +1,69 @@
+
+## What did i do?
+
+I connected my Office Apps (Word, Excel, PowerPoint, etc.) to OneDrive
+[How to add OneDrive to OfficeApps](https://support.microsoft.com/en-us/office/how-to-add-onedrive-as-a-service-3ae78a39-c0de-4c86-8abc-ad519b6de44e)
+[How to add OneDrive as additional storage](https://support.microsoft.com/en-us/office/sync-files-with-onedrive-files-on-demand-1ab17c60-af03-4c63-a24c-bb1e131f76c9)
+
+_"Why OneDrive?"_
+Because as a student, i was granted 1TB of OneDrive space
+
+_You don't like Microsoft?, here's another option:_
+[DropBox + Office Apps](https://help.dropbox.com/installs-integrations/third-party/adding-place-microsoft-office)
+[Google Drive + Office Apps](https://www.coolcatteacher.com/how-to-add-google-drive-to-microsoft-word/)
+
+---
+
+##What I feel good about?
+###1. Place-less working
+Ever since I put my important docs such as Thesis, Internship Report and other i-can't-lose document, i can do it from anywhere anytime, because it is sync in the cloud.
+For example, at home i can do my Thesis with my Desktop PC, but when i go out to somewhere i can bring my laptop along and access the work from the cloud or even accessing it from my mobile phone.
+When i store my docs in cloud, i can save it, turn-off my PC and just walk away bringing my laptop because the data is in the cloud.
+If i store it in my PC-local-disk, i should copy-paste it to my laptop first before going out, or maybe store it in a flash drive that i should carry around (more stuffs to be carried).
+
+###2. Safety(?)
+At least, physical safeness is pretty much guaranteed.
+Imagine a scenario, where your house get robbed and Cloud Data Center get robbed.
+In your house, you could lost your laptop or PC, leading to lose of data.
+In Cloud Data Center, even if they get robbed (_which is not likely to happen_), Cloud Data Center guaranteed duplication or backup of your data to fulfill the Service Level Agreement and Standard Operation Procedure and Disaster Prevention.
+This also point back to previous part about flash drive, i might lose my flash drive too, it's small-in-shape.
+Some Cloud Storage provider even provide version rollback and history of editing. If you working on document that involved several people, this feature will be useful to hunt who is making bad changes and praise the one that making good changes.
+[Version history in OneDrive](https://support.microsoft.com/en-us/office/restore-a-previous-version-of-a-file-stored-in-onedrive-159cad6d-d76e-4981-88ef-de6e96c93893)
+
+---
+
+##My Concern
+###1. I need internet
+Because the docs is in the cloud, of course you need internet to synchronize the data between your local disk and cloud. If i am about go out, i need to make sure i have internet-data to access my data in the cloud (_at least, to download it, and edit it locally, and then upload it again_).
```

Comment:
```suggestion
Since the docs are stored in the cloud, of course you will need internet access to synchronize the data between your local disk and the cloud storage. If I am about go out, I need to make sure that I have internet-data to access my data in the cloud (_at least to download it, edit it locally, and then upload it again_).
```